huaweicloud/services/lts/resource_huaweicloud_lts_transfer.go

```diff
@@ -0,0 +1,653 @@
+			"agency_domain_id":     utils.ValueIngoreEmpty(raw["agency_domain_id"]),
+			"agency_domain_name":   utils.ValueIngoreEmpty(raw["agency_domain_name"]),
+			"agency_name":          utils.ValueIngoreEmpty(raw["agency_name"]),
+			"agency_project_id":    utils.ValueIngoreEmpty(raw["agency_project_id"]),
+			"be_agency_domain_id":  domainID,
+			"be_agency_project_id": projectID,
+		}
+		return params
+	}
+	return nil
+}
+
+func buildLogTransferInfoLogTransferDetail(rawParams interface{}) map[string]interface{} {
+	if rawArray, ok := rawParams.([]interface{}); ok {
+		if len(rawArray) == 0 {
+			return nil
+		}
+		raw := rawArray[0].(map[string]interface{})
+		params := map[string]interface{}{
+			"obs_period":           utils.ValueIngoreEmpty(raw["obs_period"]),
+			"obs_period_unit":      utils.ValueIngoreEmpty(raw["obs_period_unit"]),
+			"obs_bucket_name":      utils.ValueIngoreEmpty(raw["obs_bucket_name"]),
+			"obs_transfer_path":    utils.ValueIngoreEmpty(raw["obs_transfer_path"]),
+			"obs_dir_pre_fix_name": utils.ValueIngoreEmpty(raw["obs_dir_prefix_name"]),
+			"obs_prefix_name":      utils.ValueIngoreEmpty(raw["obs_prefix_name"]),
+			"obs_eps_id":           utils.ValueIngoreEmpty(raw["obs_eps_id"]),
+			"obs_encrypted_enable": utils.ValueIngoreEmpty(raw["obs_encrypted_enable"]),
+			"obs_encrypted_id":     utils.ValueIngoreEmpty(raw["obs_encrypted_id"]),
+			"obs_time_zone":        utils.ValueIngoreEmpty(raw["obs_time_zone"]),
+			"obs_time_zone_id":     utils.ValueIngoreEmpty(raw["obs_time_zone_id"]),
+			"dis_id":               utils.ValueIngoreEmpty(raw["dis_id"]),
+			"dis_name":             utils.ValueIngoreEmpty(raw["dis_name"]),
+			"kafka_id":             utils.ValueIngoreEmpty(raw["kafka_id"]),
+			"kafka_topic":          utils.ValueIngoreEmpty(raw["kafka_topic"]),
+			"tags":                 utils.ValueIngoreEmpty(raw["delivery_tags"]),
+		}
+		return params
+	}
+	return nil
+}
+
+func resourceLtsTransferRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	var mErr *multierror.Error
+
+	// getTransfer: Query the log transfer task.
+	var (
+		getTransferHttpUrl = "v2/{project_id}/transfers"
+		getTransferProduct = "lts"
+	)
+	getTransferClient, err := cfg.NewServiceClient(getTransferProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating LTS Client: %s", err)
+	}
+
+	getTransferPath := getTransferClient.Endpoint + getTransferHttpUrl
+	getTransferPath = strings.ReplaceAll(getTransferPath, "{project_id}", getTransferClient.ProjectID)
+
+	getTransferOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+		MoreHeaders: map[string]string{"Content-Type": "application/json"},
+	}
+
+	getTransferResp, err := getTransferClient.Request("GET", getTransferPath, &getTransferOpt)
+
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving LTS Transfer")
```

Comment:
```suggestion
		fmt.Errorf("error retrieving LTS transfer: %s", err)
```
The list API may not cause 404 error. I think it can just return error.